### DISCUSSION TITLE
A React context manager

### DISCUSSION THREAD

**Discussion by rob-blackbourn** [duckdb/duckdb-wasm#1652] [Show and tell]
I've written a React context manager. You can find it [here](https://github.com/rob-blackbourn/jetblack-duckdb-react).

**Comment by bluehat974** [comment]
You should probably propose this to https://github.com/duckdb/duckdb-web or https://github.com/davidgasquez/awesome-duckdb

**Reply by rob-blackbourn** [reply]
Thanks :) I made a PR to awesome.
